src/test/java/org/cactoos/collection/NoNullsTest.java

```diff
@@ -120,4 +130,145 @@ void throwsErrorIfNullInAddAll() {
             )
         ).affirm();
     }
+
+    @Test
+    void behavesAsCollection() {
+        new Assertion<>(
+            "Must behave as collection",
```

Comment:
@andreoss I think we must improve this message to something like `Must behave as a no null collection"` to *emphasize* the no nulls capability. WDYT?